component-samples/scripts/keys_gen.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
```

Comment:
License header missing

Reply:
Added the licence header in the suggested file.